app/views/case_information/show.html.erb

```diff
@@ -16,13 +16,6 @@
 
 <h1 class="govuk-heading-xl govuk-!-margin-top-4 govuk-!-margin-bottom-4">Case information</h1>
 
-<p class="govuk-!-margin-bottom-8">
-  <% if @delius_data.any? %>
-    Case information last updated from nDelius: <%= format_date(@delius_data.last.updated_at) %>
-  <% end %>
-  Next scheduled update: <%= format_date(@next_update_date) %>
-</p>
-
```

Comment:
I wonder if this is still a useful piece of information for users to see.

We're still updating our view of nDelius as a polling/batch job – rather than being live. So it still stands that there's a "last updated" and "next scheduled update".

Would be good to know what to do here. Is there a user need to keep it?

Reply:
We've lost the ability to know when case information was last updated - although I guess the case_information record updated_at gives some clues...?

Reply:
@starswan Yes – we don't have that as it stands. But I'm thinking it should be possible for us to implement something which touches a _"last seen in nDelius"_ field every night, even if the record is already 'in sync'.

But that's only worth doing if we think there's still a design / user need for it.

Reply:
I've restored the feature for now using last touched for case information which is *nearly* right